src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -205,10 +205,11 @@ export default {
       // v-slot attributes are lower-cased by the browser.
       // Switched to round bracket syntax to prevent confusion with
       // dynamic slot name syntax.
-      const slotNames = [`cell(${key})`, `cell(${key.toLowerCase()})`, 'cell()']
-      let $childNodes = this.hasNormalizedSlot(slotNames)
-        ? this.normalizeSlot(slotNames, slotScope)
-        : toString(formatted)
+      // const slotNames = [`cell(${key})`, `cell(${key.toLowerCase()})`, 'cell()']
```

Comment:
@tmorehouse What about this commented out line?

Reply:
I just left it in for reference to the slot names we look for.

Reply:
Cold be re-worded as a true comment

Reply:
Changed it to a descriptive comment